src/main/java/edu/oregonstate/cs361/battleship/Game.java

```diff
@@ -0,0 +1,760 @@
+            temp.add(new Coord(3,9));temp.add(new Coord(3,10));temp.add(new Coord(2,9));
+            temp.add(new Coord(2,10));
+
+            model.setPlayerHits(temp);
+
+
+            //MAKES THE L
+            temp = new ArrayList<Coord>();
+
+            temp.add(new Coord(2,4));temp.add(new Coord(2,5));temp.add(new Coord(3,4));
+            temp.add(new Coord(3,5));temp.add(new Coord(4,4));temp.add(new Coord(4,5));
+            temp.add(new Coord(5,4));temp.add(new Coord(5,5));temp.add(new Coord(6,4));
+            temp.add(new Coord(6,5));temp.add(new Coord(7,4));temp.add(new Coord(7,5));
+            temp.add(new Coord(8,4));temp.add(new Coord(8,5));temp.add(new Coord(8,6));
+            temp.add(new Coord(8,7));temp.add(new Coord(8,8));temp.add(new Coord(9,4));
+            temp.add(new Coord(9,5));temp.add(new Coord(9,6));temp.add(new Coord(9,7));
+            temp.add(new Coord(9,8));
+
+            model.setComputerHits(temp);
+
+        }
+        else
+        {
+            //Add in the coords for the W and L for the winner and loser
+            //MAKES THE W
+            temp.add(new Coord(2,1));temp.add(new Coord(2,2));temp.add(new Coord(3,1));
+            temp.add(new Coord(3,2));temp.add(new Coord(4,1));temp.add(new Coord(4,2));
+            temp.add(new Coord(4,3));temp.add(new Coord(5,2));temp.add(new Coord(5,3));
+            temp.add(new Coord(6,2));temp.add(new Coord(6,3));temp.add(new Coord(6,4));
+            temp.add(new Coord(7,3));temp.add(new Coord(7,4));temp.add(new Coord(7,5));
+            temp.add(new Coord(8,4));temp.add(new Coord(8,5));temp.add(new Coord(7,6));
+            temp.add(new Coord(6,6));temp.add(new Coord(7,7));temp.add(new Coord(8,7));
+            temp.add(new Coord(8,7));temp.add(new Coord(7,8));temp.add(new Coord(6,8));
+            temp.add(new Coord(6,9));temp.add(new Coord(5,8));temp.add(new Coord(5,9));
+            temp.add(new Coord(4,8));temp.add(new Coord(4,9));temp.add(new Coord(4,10));
+            temp.add(new Coord(3,9));temp.add(new Coord(3,10));temp.add(new Coord(2,9));
+            temp.add(new Coord(2,10));
+
+            model.setComputerHits(temp);
+
+
+            //MAKES THE L
+            temp = new ArrayList<Coord>();
+
+            temp.add(new Coord(2,4));temp.add(new Coord(2,5));temp.add(new Coord(3,4));
+            temp.add(new Coord(3,5));temp.add(new Coord(4,4));temp.add(new Coord(4,5));
+            temp.add(new Coord(5,4));temp.add(new Coord(5,5));temp.add(new Coord(6,4));
+            temp.add(new Coord(6,5));temp.add(new Coord(7,4));temp.add(new Coord(7,5));
+            temp.add(new Coord(8,4));temp.add(new Coord(8,5));temp.add(new Coord(8,6));
+            temp.add(new Coord(8,7));temp.add(new Coord(8,8));temp.add(new Coord(9,4));
+            temp.add(new Coord(9,5));temp.add(new Coord(9,6));temp.add(new Coord(9,7));
+            temp.add(new Coord(9,8));
+
+            model.setPlayerHits(temp);
+        }
+    }
+}
+
```

Comment:
Correctly implemented version of the old Main class as an object oriented Game class.  Allows for easier understanding of the structure of Main and the order in which functions are executed.